project_one/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Project1</title>
+    <link rel="stylesheet" type="text/css" href="index.css" />
+  </head>
+  <body>
+    <header id="main-header">
+      <div class="container">
+        <h1>Project1</h1>
+      </div>
+    </header>
+
+    <nav id="navbar">
+      <div class="container">
+        <ul>
+          <li><a href="#Home">Home</a></li>
+          <!-- <li><a href="my_calc.html">Calculator Link</a></li> -->
+        </ul>
+      </div>
+    </nav>
+
+    <section>
+      <div class="container" id="receiver-div">
+        <p>This is section one:</p>
+        <button onclick="myFunction()" id="grading-element-one">Button1</button>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <p>This is section two:</p>
+        <a id="grading-element-two" href="about.html">About</a>
+      </div>
+    </section>
+
+    <section>
+      <div class="container" id="grading-element-three">
+        <p>This is section three:</p>
+        <button onclick="myFunction2()">Button3</button>
+        <div id="grading-element-three-consequence" hidden>
+          awesome class
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container form">
+        <p>This is section four:</p>
+        <form
```

Comment:
when i post the form the data sent is `truthyVal=testOne&text=`. It needs to be `testOne=(something)&testTwo=(something)&testThree=(something)`